src/european_values/generative_training.py

```diff
@@ -19,7 +22,7 @@ def train_generative_model(
     """Train a Gaussian Mixture Model on EU survey data.
 
     Args:
-        survey_df: Survey data DataFrame
+        survey_df: Survey data DataFrame (already processed and subset-filtered)
```

Comment:
Adding some clarification about non-normalisation. Removed the "processed" to avoid confusion with whether processing includes normalisation.

```suggestion
        survey_df: A non-normalised dataframe containing the survey responses.
```